devicetypes/smartthings/zll-dimmer-bulb.src/zll-dimmer-bulb.groovy

```diff
@@ -13,13 +13,12 @@
  */
 import groovy.transform.Field
 
-@Field Boolean hasConfiguredHealthCheck = false             
+@Field Boolean hasConfiguredHealthCheck = false
 
 metadata {
 	definition(name: "ZLL Dimmer Bulb", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "oic.d.light", runLocally: true, minHubCoreVersion: '000.021.00001', executeCommandsLocally: true) {
 		capability "Actuator"
 		capability "Configuration"
-		capability "Polling"
```

Comment:
You should be able to remove the `poll` command from all these DTHs too. I'm checking on the details of how to remove the capability and ensure that `poll` stops being called.